tests/orgs/test_views.py

```diff
@@ -44,10 +45,10 @@ def test_organization_members_tab(app: DjangoTestApp, data_for_tabs):
     assert list(resp.html.find("li", class_="is-active").a.stripped_strings) == ["Organizacijos nariai"]
 
 
-@pytest.mark.django_db
+@pytest.mark.haystack
 def test_organization_dataset_tab(app: DjangoTestApp, data_for_tabs):
```

Comment:
```suggestion
def test_organization_dataset_tab(app: DjangoTestApp, data_for_tabs):
```

Is suggest, to avoid reusing data fixtures, better user factories every time, because most of the time, you need different data. Like in this, case, in order to reproduce [the issue](https://github.com/atviriduomenys/katalogas/pull/170#discussion_r989791578), you need to add at least two datasets, each with different organization and then use the search, to check if only one is shown.